test/sandbox/app/components/slots_v2_without_content_block_component.rb

```diff
@@ -4,7 +4,7 @@ class SlotsV2WithoutContentBlockComponent < ViewComponent::Base
   renders_one :title, "MyTitleComponent"
 
   class MyTitleComponent < ViewComponent::Base
-    def initialize(title:)
+    def initialize(title)
```

Comment:
This is a somewhat unrelated change, but having a kwarg called `title` for a method called `title` didn't seem great, but it also caused some issues with delegation.